test/support/factories/alerts/alert.ex

```diff
@@ -28,4 +28,38 @@ defmodule Test.Support.Factories.Alerts.Alert do
       url: Faker.Internet.url()
     }
   end
+
+  def alert_for_route_factory(attrs) do
+    {route_id, attrs} = Map.pop(attrs, :route_id)
+
+    build(
+      :alert,
+      attrs
+      |> Map.put(
+        :informed_entity,
+        InformedEntitySet.build(:informed_entity_set,
+          entities: [InformedEntity.build(:informed_entity, route: route_id)]
+        )
+      )
+    )
+  end
+
+  def active_during(alert, time) do
```

Comment:
Is it intentional for it to always output within-a-day active periods, as opposed to ones that span many days? If so, this is better renamed to `active_day` or `active_within_day` IMO.

Reply:
Not really - I basically wanted to be able to say "Gimme an alert that's active at `time`", but I don't think it matters a whole lot exactly when.